src/jobservice/job/impl/gc/garbage_collection_test.go

```diff
@@ -378,6 +380,14 @@ func (suite *gcTestSuite) TestSweep() {
 	suite.Nil(gc.sweep(ctx))
 }
 
+func (suite *gcTestSuite) TestSaveRes() {
+	ctx := &mockjobservice.MockJobContext{}
+	logger := &mockjobservice.MockJobLogger{}
+	ctx.On("GetLogger").Return(logger)
+	mock.OnAnything(ctx, "Checkin").Return(nil)
+	suite.Nil(saveGCRes(ctx, 123456, 100, 100))
```

Comment:
maybe the parameters `100, 100` are just mock data representing `blobs` & `manifests` respectively. Can they be equal? or blob number should normally larger than manifests number.